packages/app/background-jobs-common/UPGRADING.md

```diff
@@ -0,0 +1,45 @@
+# Upgrading Guide
+
+## Upgrading from `4.0.0` to `5.0.0`
+
+### Description of Breaking Changes
+
+1. **Property Removal**
+  - The deprecated `processCalls` property from `FakeBackgroundJobProcessor` has been removed.
+
+2. **Method Changes in `AbstractBackgroundJobProcessor`**
+  - The method `getActiveQueueIds` has been removed. A new constant method now serves this purpose.
```

Comment:
`A new constant method now serves this purpose.` -> this sounds unclear, which constant method does this refer to?

Reply:
ok, I see there are instructions further in the doc. maybe it's worth mentioning that for details check the migration steps?

Reply:
Forgot to paste the name, sorry 😓

Reply:
Actually it is described but on migration steps, this part is like a summary of the breaking changes, do you think we should also mention it here?

Reply:
mentioning name or mentioning that you should check the migration steps should suffice